pkgs/games/cataclysm-dda/bn/default.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  mkCataclysm,
+  fetchFromGitHub,
+  sqlite,
+}:
+
+mkCataclysm (finalAttrs: {
+  pname = "cataclysm-bn";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "cataclysmbnteam";
+    repo = "Cataclysm-BN";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-OKxYJKeSPGus9zbRsCbNIXx+qc27Ae/Se6oeFAGMHUY=";
+  };
+
+  versionFile = ''
+    build type: nixpkgs
+    build number: ${finalAttrs.version}
+    commit sha: ${finalAttrs.src.rev}
+    commit url: ${finalAttrs.src.resolvedUrl}
+  '';
+
+  postPatch = ''
+    # Remove broken symlinks
+    rm data/json/external_tileset/README.md
+    rm data/json/mapgen/lab/README.md
+
+    # Fix build failure since git is not used
+    echo "$versionFile" > VERSION.txt
+  '';
+
+  useCmake = true;
+
+  buildInputs = [
+    sqlite
+  ];
+
+  meta = {
+    homepage = "https://docs.cataclysmbn.org/";
+    changelog = "https://github.com/cataclysmbnteam/Cataclysm-BN/releases/tag/v${finalAttrs.version}";
+    description = "roguelike with sci-fi elements set in a post-apocalyptic world";
+    longDescription = ''
+      Cataclysm: Bright Nights is a roguelike with sci-fi elements set in a post-apocalyptic world.
+
+      While some have described it as a "zombie game", there is far more to Cataclysm than that.
+      Struggle to survive in a harsh, persistent, procedurally generated world. Scavenge the remnants
+      of a dead civilization for food, equipment, or, if you are lucky, a vehicle with a full tank of
+      gas to get you the hell out of there.
+
+      Fight to defeat or escape from a wide variety of powerful monstrosities, from zombies to giant
+      insects to killer robots and things far stranger and deadlier, and against the others like yourself,
+      who want what you have.
+
+      Find a way to stop the Cataclysm ... or become one of its strongest monsters.
```

Comment:
Is this from the Bright Nights README? If so, we probably can't use it, since it'd be CC-BY-SA.